code/modules/client/client_procs.dm

```diff
@@ -169,6 +169,27 @@ GLOBAL_VAR_INIT(external_rsc_url, TRUE)
 
 	var/full_version = "[byond_version].[byond_build ? byond_build : "xxx"]"
 
+	if(CONFIG_GET(flag/log_access))
+		for(var/I in GLOB.clients)
+			if(!I || I == src)
```

Comment:
Could you trace stack null vars in `GLOB.clients`?

Reply:
They will never be there otherwise may as well add it to all for loops

Reply:
But yeah it's /tg/code so why not

Reply:
Could just remove the check as well.